scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2420,7 +2472,7 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
     val isMutable = token.is[`var`]
     next()
     val lhs: List[Pat] = commaSeparated(noSeq.pattern2().require[Pat]).map {
-      // TODO: val `x`: Int should be parsed as Decl.Val(..., Pat.Var.Term(Term.Name("x")), ...)
```

Comment:
Because that todo is done.
```
scala> q"val `x`: Int".show[Raw]
res2: String = Decl.Val(Nil, List(Pat.Var.Term(Term.Name("x"))), Type.Name("Int"))
```